### ISSUE TITLE
Allow user to add custom cookie for site easily

### ISSUE THREAD

**Issue by juhiyu** [vanowm/FirefoxCookiesManagerPlus#86]
Hellos, after using CM+ one feature I think would be useful is to option my own custom cookies to site easily.

What I mean is I can save custom cookies, example: session=true, product_id=22938190168, logged_in=1 etc
So if I want to use it on my website.com, I visit it, click a dropdown maybe called "Custom Cookies" from CM+ to insert one or many of my custom cookies without having to manually type in each one each time.

I don't know if I'm a "power user" so this feature may only help more power users for when they are testing how a production site works with some cookies, clearing them and adding (certain of) them back without manually writing each out each one each time. may also be other use cases i am not thinking

**Comment by vanowm** [comment]
Why not just create the cookies you need, then back them up and restore when you need them?
If I understand your request correctly it would be the same as using text editor and create cookie file manually and then just drag'n drop it into CM+ window (minus the drag'n drop part)

**Comment by juhiyu** [comment]
Yes, idea is similar to restoring cookies and then manually keeping or deleting cookies as needed (but to add back cookies as it is now is manually and custom-saved cookies will able to be applied to any site user wants not just imported cookies for the 1 site it is defined for). This is mainly for time saving less clicks and actions , ease of use when doing it many times
